project/frontend/src/containers/CreateCompetition/CreateCompetition.js

```diff
@@ -1,68 +1,45 @@
-import React from "react"
-import classes from "./CreateCompetition.module.css"
-import {Link} from "react-router-dom"
+import React from "react";
+import classes from "./CreateCompetition.module.css";
+import { Link } from "react-router-dom";
+import LinkButton from "../../components/UI/LinkButton/LinkButton";
+import ParticipantCard from "../../components/ParticipantCard/ParticipantCard";
 
 const CreateCompetition = (props) => {
   return (
-    <div className={classes.CreateCompetitionPage}>
-      <div className={classes.Title}>
-        <p>Create a competition</p>
-        <br></br>
-      </div>
+    <div className={classes.CreateCompetitionContainer}>
+      <p>Create a competition</p>
       <div className={classes.MainBox}>
         <div className={classes.CompetitionInfoLeft}>
           <div className={classes.TextColumnContainer}>
-            <div>
-              <span>Competition Name</span>
-              <br></br>
-              <br></br>
-            </div>
-            <div>
-              <span>Start Date</span>
-              <br></br>
-              <br></br>
-              <br></br>
-              <br></br>
-            </div>
-            <div>
-              <span>End Date</span>
-              <br></br>
-              <br></br>
-              <br></br>
-              <br></br>
-            </div>
+            <span>Competition Name</span>
+            <span>Start Date</span>
+            <span>End Date</span>
           </div>
           <div className={classes.BoxColumnContainer}>
-            <br></br>
-            <br></br>
             <input type="text" />
-            <br></br>
-            <br></br>
             <input type="date" />
-            <br></br>
-            <br></br>
-            <br></br>
-            <br></br>
             <input type="date" />
-            <br></br>
-            <br></br>
-            <br></br>
-            <br></br>
           </div>
         </div>
-        <div className={classes.CompetitiomInfoRight}>
+        <div className={classes.CompetitionInfoRight}>
           <p>Participants</p>
+          <div className={classes.AddParticipants}>
+            <p>
+              {" "}
+              Add Participants: <input type="text" />{" "}
+              <LinkButton inverted="true">Add</LinkButton>
```

Comment:
Why is this a `LinkButton`? Adding a participant shouldn't change page history, right?

Reply:
No it doesn't change the page it just display the email on the space below. I'm only using it cause the style match the others but I can definitely create my own button if that will make it more efficient.

Reply:
Should be fine, just a naming thing.